server/api/controllers/sermons/get-all-announcements.js

```diff
@@ -0,0 +1,48 @@
+const { DateTime } = require('luxon');
+const he = require('he');
+
+const transformAnnouncement = (announce) => {
```

Comment:
it'd be good to look into some of the ways we're using caching and helpers to do a lot of this. can talk to you more about it thru call and help walk through it with you

Reply:
also maybe `annoucement` instead of `announce`